examples/basis_analysis.jl

```diff
@@ -41,18 +41,24 @@ end
 L = 12
 sites = siteinds("S=1/2", L)
 H = xxz_chain(sites; cutoff=1e-9);
-dm = DMRG(; n_states=1, tol_degeneracy=0.0, sweeps=default_sweeps(; cutoff_max=1e-9),
-          observer=() -> DMRGObserver(; energy_tol=1e-9));
+dm = DMRG();
 Δ = range(-1.0, 2.5; length=40)
 hJ = range(0.0, 3.5; length=40)
 grid_train = RegularGrid(Δ, hJ);
 
 # Up to this points we chose reasonable parameters for the basis assembly. However, we now
-# intend to generate a bad surrogate with purposefully bad settings. Namely, we limit our
-# basis size to `n_truth_max=6` truth solves and use a compression cutoff  of
-# `cutoff=1e-10` which is lower than the energy and singular value cutoff above:
+# intend to generate a bad surrogate with purposefully bad settings.
+#
+# !!! warning "Bad settings"
+#     The offline settings that are chosen here and also later in the example are designed
+#     to produce wrong and unconverged results, i.e. the "side effects" observed in this
```

Comment:
Better:

are intentionally chosen to produce unconverged (and thus wrong) results